src/tempfile.rs

```diff
@@ -142,16 +130,16 @@ mod tests {
             (to_check >= lower) && (to_check <= upper)
         }
 
-        let tempname = "/tmp/asdf";
-        let t = TempFile::new_with_prefix(tempname).unwrap();
-        assert_eq!(tempname, "/tmp/asdf");
+        let mut prefix = temp_dir();
+        prefix.push("asdf");
```

Comment:
Why do you need this push?

Reply:
Just wanted to keep the previous behavior, namely to use the prefix `/tmp/asdf` (on Linux).